src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1509,6 +1509,15 @@
   <data name="DoNotGuardDictionaryRemoveByContainsKeyTitle" xml:space="preserve">
     <value>Unnecessary call to 'Dictionary.ContainsKey(key)'</value>
   </data>
+  <data name="DoNotGuardSetAddOrRemoveByContainsDescription" xml:space="preserve">
+    <value>Do not guard 'Add(item)' or 'Remove(item)' with 'Contains(item)' for the set. The former two already check whether the item exists and will return if it was added or removed.</value>
+  </data>
+  <data name="DoNotGuardSetAddOrRemoveByContainsMessage" xml:space="preserve">
+    <value>Do not guard '{0}' with '{1}'</value>
+  </data>
+  <data name="DoNotGuardSetAddOrRemoveByContainsTitle" xml:space="preserve">
+    <value>Remove unnecessary call</value>
```

Comment:
@mpidash it seems you misunderstood my [comment](https://github.com/dotnet/roslyn-analyzers/pull/6767#discussion_r1267102149) `Showing same title in the fixer do not look good, it should be something that illustrates what the fixer does, like the one below. It looks, you can even use the below RemoveRedundantGuardCallCodeFixTitle as is for the fixer` 

I meant to use different title for analyzer and code fixer. The analyzer title can be kept as it was `Unnecessary call to 'Set.Contains(item)'` or omit Set: `Unnecessary call to 'Contains(item)'`. Then the code fixer should be illustrate the action it is fixing i.e.: `Remove unnecessary call` for that don't need to create another title with same value, can just use the `RemoveRedundantGuardCallCodeFixTitle` below

Reply:
Oh, sorry about that and thanks for the clarification. I have dropped the `Set` in the message as you suggested to be consistent with `DoNotGuardSetAddOrRemoveByContainsDescription`.

Changed in 5fef8d0.